.github/workflows/osv-scanner.yml

```
@@ -33,6 +33,12 @@
         -r
         --skip-git
         ./
+        steps:
+                - name: Upload scan results
+                  uses: actions/upload-artifact@v3
+                  with:
+                    name: scan-results
+                    path: ./scan-results 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)